src/cms/templates/pages/page_tree_node.html

```diff
@@ -23,7 +23,7 @@
         <span class="collapse-subpages cursor-pointer inline-block align-middle"
               title="{% trans 'Collapse all subpages' %}"
               data-page-id="{{ page.id }}"
-              data-page-children="{{ page|get_children }}">
+              data-page-children="{{ children_in_filter }}">
```

Comment:
This template tag is not required, because the `get_children` filter will only return the filtered children, too.
But in this case where we have a partial `TreeQuerySet` only, the `get_children` filter is buggy unfortunately.
So we should fix that over here: https://github.com/Integreat/integreat-cms/blob/38c0601eb4cc94f652fc40d1933adc530d74cab8/src/cms/templatetags/tree_filters.py#L35
This line should actually be:
```
return [child.id for child in node.get_children()]
```
Then the filter should work with partial trees, too.